src/executables/sysrepo-plugind.c

```diff
@@ -480,6 +486,10 @@ main(int argc, char **argv)
         case 'd':
             debug = 1;
             break;
+        case 'p':
+            pidfile = strdup(optarg);
```

Comment:
There's no matching call to `free(pidfile)`, so this leaks memory.

Reply:
I was going to rely on the OS to reclaim memory, but I can see memory checking tools report it.